utils/cache.js

```diff
@@ -43,10 +43,11 @@ class Cache {
    * @return {Object} value
    */
   get(key) {
-    const debounced = this.deleteTriggers.get(key);
-    debounced(this.map);
-    this.deleteTriggers.set(key, debounced);
-    return this.map.get(key);
+    if (this.has(key)) {
+      const debounced = this.deleteTriggers.get(key);
+      debounced(this.map);
```

Comment:
`debounced` now takes two parameters. Please add a test for this case (initially set, then get, then expires).